src/api/controllers/lines.js

```diff
@@ -6,13 +6,13 @@ import response from './response.js';
 
 // get /lines
 function getLineByAuthUser(req, res) {
-
-    Line.findOne({user_id: req.user._id}).exec(function (err, line) {
+    Line.find( { user_id: req.user._id } ).exec(function (err, lines) {
         if (err)
             return res.send(err);
-        res.status(200).json(line);
+        res.status(200).json({
+            "lines": lines
```

Comment:
hey change this to /lines/auth. We may not even need /lines, and if we do, it'll be a admin only search.

Reply:
putting /lines in a ticket then